packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -310,8 +311,21 @@ export class Viewer {
     await this.unload();
```

Comment:
`unload` should only be called if we initiate a new WS connection.

Reply:
This is handled within the function. It checks if there is a streamDisposable present, otherwise does nothing.